packages/repository/README.md

```diff
@@ -238,62 +31,50 @@ The repository module provides APIs to define LoopBack 3.x data sources and
 models. For example,
 
 ```ts
-import {
-  DataSourceConstructor,
-  juggler,
-  Entity,
-  model,
-  ModelDefinition
-} from '@loopback/repository';
+// src/datasources/db.datasource.ts
+import {juggler, DataSourceConstructor} from '@loopback/repository';
 
-export const ds: juggler.DataSource = new DataSourceConstructor({
+export const db: juggler.DataSource = new DataSourceConstructor({
   name: 'db',
   connector: 'memory',
 });
+```
+
+```ts
+// src/models/note.model.ts
+import {model, Entity, property} from '@loopback/repository';
 
 @model()
 export class Note extends Entity {
-  static definition = new ModelDefinition({
-    name: 'note',
-    properties: {
-      id: {name: 'id', type: 'number', id: true},
-      title: 'string',
-      content: 'string'
-    }
-  })
-};
+  @property({id: true})
+  id: string;
+  @property() title: string;
+  @property() content: string;
+}
 ```
 
 **NOTE**: There is no declarative support for data source and model yet in
```

Comment:
I thought we have declarative support for data source.  
In db.datasource.ts, we can do something like below:
```
const dsConfigPath = path.resolve('config', 'datasources.json');
const config = require(dsConfigPath);
export const db = new DataSourceConstructor(config);
```

Reply:
@dhmlau I personally thought that the meaning of having declarative support meant that no config is necessary outside of the declarative file itself (the declarative file being config.json). Please correct me if I am wrong.

Reply:
I was thinking that we can still define the datasource.json like what we did in LB3.  It's not the ultimate goal for declarative support.  So i'm not sure what we wanna say about this statement. Maybe @raymondfeng can help :)